app/src/main/java/com/amaze/filemanager/utils/Utils.java

```diff
@@ -241,8 +262,8 @@ public static int nthToLastCharIndex(int elementNumber, String str, char element
      * Formats input to plain mm:ss format
      */
     public static String formatTimer(long timer) {
```

Comment:
Nowhere does it say the timer is in seconds, I think that is what led to the original mistake.